releasenotes/add-verified-field-to-analysis-results-5926fa1428ef1df5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    :class:``qiskit.providers.ibmq.analysis_result.AnalysisResult` now has an additional
+    ``verified`` attribute which identifies if the ``quality`` has been verified by a human.
```

Comment:
```suggestion
    ``verified`` attribute which identifies if the ``quality`` has been verified by a human.

```

nit: I don't know if you really need the new line at the end of the file or not, but GitHub is highlighting it.